xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedAttachments.xml

```diff
@@ -145,6 +145,7 @@ $response.setContentType('application/json')
 #end
 #set($filenameFilter = $request.get('datt.filename'))
 #set($docNameFilter = $request.get('datt.docName'))
+#set($dateFilter = $request.get('ddoc.date'))
```

Comment:
The request parameter you read the filter value from must correspond to a live table column. Look for the ``$columns`` variable above. There's no ``ddoc.date`` column listed there. There is an ``datt.date`` column though (because this live table lists deleted attachments not deleted documents). It looks like you made the pull request without testing your changes, which is very bad.